db/schema.rb

```diff
@@ -210,6 +210,8 @@
     t.string "course_length"
     t.string "description"
     t.integer "accredited_provider_id"
+    t.jsonb "subject_codes"
+    t.string "funding_type"
```

Comment:
Is this different from `financial_support`?

Reply:
funding type is apprenticeship, salaried or fee. Financial support details bursaries etc.

Reply:
Thanks @davidgisbey!